Hearthstone Deck Tracker/Controls/Overlay/Battlegrounds/HeroPicking/BattlegroundsHeroPicking.xaml

```diff
@@ -19,29 +19,43 @@
             <Setter Property="LayoutTransform" Value="{Binding LayoutTransform, RelativeSource={RelativeSource AncestorType=heroPicking:BattlegroundsHeroPicking}}"/>
         </Style>
     </UserControl.Resources>
-    <Grid>
-        <!-- Centered over heroes -->
-        <Grid Margin="14,15,0,0" HorizontalAlignment="Center" VerticalAlignment="Center"
-             anim:FadeAnimation.Visibility="{Binding Visibility}" Visibility="Collapsed"
-             anim:FadeAnimation.Direction="Down" anim:FadeAnimation.Distance="20" anim:FadeAnimation.Duration="0:0:0.2"
-        >
-            <ItemsControl ItemsSource="{Binding HeroStats}" Margin="-16,0">
-                <ItemsControl.ItemTemplate>
-                    <DataTemplate>
-                        <Grid Height="480" Width="266" Margin="16,0">
-                            <heroPicking:BattlegroundsSingleHeroStats
-                                SetSelectedHeroDbfIdCommand="{Binding SetSelectedHeroDbfIdCommand, RelativeSource={RelativeSource AncestorType=heroPicking:BattlegroundsHeroPicking}}"
-                            />
-                        </Grid>
-                    </DataTemplate>
-                </ItemsControl.ItemTemplate>
-                <ItemsControl.ItemsPanel>
-                    <ItemsPanelTemplate>
-                        <UniformGrid Rows="1"/>
-                    </ItemsPanelTemplate>
-                </ItemsControl.ItemsPanel>
-            </ItemsControl>
+    <Grid Visibility="{Binding Visibility}">
+        <Grid anim:FadeAnimation.Visibility="{Binding StatsVisibility}" Visibility="Collapsed"
+              anim:FadeAnimation.Direction="Down" anim:FadeAnimation.Distance="20" anim:FadeAnimation.Duration="0:0:0.2">
```

Comment:
Are there any edge cases here where it doesn't correctly play the animation because `Visibility` was `!= Visible` for longer than `StatsVisibility`?

Reply:
Code updated moving the `Visibility` binding to the visibility button only that also works and makes it not affect the animation that uses `StatsVisibility`.